frontend/src/metabase/visualizations/echarts/cartesian/option/series.ts

```diff
@@ -519,7 +519,7 @@ export const buildEChartsSeries = (
           return buildEChartsLineAreaSeries(
             seriesModel,
             seriesSettings,
-            chartModel.dataset,
+            chartModel.transformedDataset,
```

Comment:
Not related to the main issue, but I noticed we weren't using `transformedDataset` here so I changed it